src/log/log.go

```diff
@@ -0,0 +1,34 @@
+package logger
+
+import (
+	"fmt"
+	"log"
+)
+
+const (
+	// ANSI color codes
+	Red    = "\033[31m"
+	Green  = "\033[32m"
+	Yellow = "\033[33m"
+	White  = "\033[97m"
+	Reset  = "\033[0m"
+)
+
+func Printf(color, format string, v ...interface{}) {
```

Comment:
Inconsistent variable naming. Here it is `v`, in the next two functions it is `args`.